css/style.css

```diff
@@ -0,0 +1,101 @@
+:root{
+    --color-white: #f3f3f3;
+    --color-blue: #23395d;
+    --color-greeen:#37af65;
+}
+
+*::before,*::after{
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 1rem;
+    font-weight: 400;
+    color: white;
+    height: 100%;
+    width: 100%;
+    background: var(--color-blue);
+    background-image:linear-gradient(115deg, rgba(58, 58, 158, 0.8),
+    rgba(136, 136, 206, 0.8)), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-repeat: no-repeat;
+    justify-content: center;
+}
+ form {
+    background: var(--color-blue);
+    opacity: .8;
+ }
+
+
+.container{
+    max-width: 768px;
+} .age{
+    margin-left: .27rem;
+    font-size: .88rem;
+    color: #e4e4e4;
+}
+
+
+
+
```

Comment:
I think these extra space makes your code not clean and it will cause linters error as well.